tests/unit/test_job_retry.py

```diff
@@ -22,6 +22,10 @@
 import google.api_core.retry
 import freezegun
 
+from google.cloud.bigquery.client import Client
+from google.cloud.bigquery import _job_helpers
+from google.cloud.bigquery.retry import DEFAULT_JOB_RETRY
```

Comment:
Import modules not classes / methods.

https://google.github.io/styleguide/pyguide.html#22-imports